src/Network/Network.Test/ScenarioTests/Generated/_metadata.txt

```diff
@@ -1,6 +0,0 @@
-ApplicationSecurityGroupTestsGenerated.cs	299A1E5866F1F33032F70699B1B9A1790689CD35
```

Comment:
Why is this file deleted? They use a generator to create some of their tests. I'm assuming this file is used by their generator.

Reply:
I agreed these changes with the Network team

Reply:
This captures the commit for the source script they used to generate the tests, so we should leave it, I think

Reply:
@EvgenyAgafonchikov  Can you confirm you need the metadata?

Reply:
I discussed this offline. 
> AFAIK PS team plans to use their own cmdlets generator, so we are dropping support of ours. In this context removing of this file is okay.

Reply:
That doesn't apply until we are actually using a new generator.